.redhat-ci.Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM fedora:24
+MAINTAINER Jonathan Lebon <jlebon@redhat.com>
+
+# NB: we also install python2 reqs here (which the builddep
+# does not catch since on F24 we build for py3) so that we
+# can do pylint for both py2 and py3 as well as reuse the
+# same image for RHEL & CentOS tests rather than maintaining
+# two separate images.
+
+RUN dnf install -y \
+        make \
```

Comment:
AKA `@buildsys-build`, as mock uses for Fedora in e.g. `config_opts['chroot_setup_cmd'] = 'install @buildsys-build'`